Autopilot/CMakeLists.txt

```diff
@@ -113,6 +113,7 @@ elseif(${KIND_OF_BUILD} STREQUAL "UNIT_TESTS")
     ${CMAKE_CURRENT_SOURCE_DIR}/Test/Inc
```

Comment:
Your'e gonna have to add the library to the target and simulation builds as well. See how that's done with the FREERTOS library.

Reply:
I'm not sure if I'm doing this correctly, but I added the datastructures folder to the Libraries section.

Reply:
that should be correct. The way to confirm is to just try to instantiate your vector class in some random file and seeing if things compile in the target build.